chcemvediet/apps/__init__.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+import django.db.backends.sqlite3.base
+from django.apps import AppConfig
+from django.db.backends import utils as backend_utils
+
+
+class ChcemvedietConfig(AppConfig):
+    name = u'chcemvediet.apps'
+
+    def ready(self):
+        django.db.backends.sqlite3.base._sqlite_format_dtdelta = self._sqlite_format_dtdelta
```

Comment:
Pozor, workarround mozeme aplikovat iba pri developovani na localhoste. V produkcii ho nesmieme aplikovat, kedze sa tyka iba sqlite. Pridaj podmienku `if settings.DATABASES['default']['ENGINE'] == 'django.db.backends.sqlite3'` alebo take nieco.

Potom tiez importy zo zaciatku suboru presun sem do `ready` do vnutra podmienky. Nech sa `sqlite3` importne iba, ked pouzivame `sqlite3`. Aby sa to neimpotovalo v produkcii.

Reply:
:ok: